### DISCUSSION TITLE
Convert bootstrap forms to tailwindcss

### DISCUSSION THREAD

**Discussion by mlieshout** [tailwindlabs/tailwindcss#2745] [Help]
Hi,

I need to convert a couple of bootstrap forms to Tailwind CSS. 
What is the best way to do this? Using TtailwindUI or Tailwind CSS Custom Forms?

**Comment by majortom64** [comment]
I think it would depend on whether you already see a TailwindUI form to do what you need as a base or if you are just building it from scratch. Tailwind CSS Custom Forms are just a set of base styles that make forms look a bit better. Tailwind UI is a set of tailwind based templates that one can purchase as the basis for a specific design.
